ivy/functional/backends/jax/gradients.py

```diff
@@ -145,10 +145,15 @@ def stop_gradient(
 
 
 def jac(func: Callable):
-    grad_fn = lambda x_in: ivy.to_native(func(x_in), nested=True)
+    grad_fn = lambda x_in: ivy.to_native(
+        func(ivy.to_ivy(x_in, nested=True)),
+        nested=True,
+        include_derived={tuple: True},
+    )
     callback_fn = lambda x_in: ivy.to_ivy(
-        jax.jacfwd(grad_fn)((ivy.to_native(x_in, nested=True))),
+        jax.jacrev(grad_fn)((ivy.to_native(x_in, nested=True))),
```

Comment:
I think there's some cases where `jacrev` is consistent with the other backends and others where `jacfwd` is more consistent. I had also created an [issue](https://github.com/google/jax/issues/14149) for this, but doesn't seem to have been responded to.

Reply:
It's a problem with torch.jacfwd as well. I only tried with the `eigh` function though. It doesn't return nan but returns different values in each case.

Reply:
oh okay got it! Thanks for clarifying, we can replace `jacrev` by `jacfwd` if it's more consistent for the cases we're testing 👍